src/parser/view_transform.c

```diff
@@ -4610,7 +4656,9 @@ mq_check_rewrite_select (PARSER_CONTEXT * parser, PT_NODE * select_statement)
     }
 
   from = select_statement->info.query.q.select.from;
-  if (from && (from->next || pt_has_aggregate (parser, select_statement)))
+  if (from
+      && (from->next || (from->info.spec.derived_table_type == PT_DERIVED_DBLINK_TABLE)
```

Comment:
This condition doesn't seem to be necessary.

Reply:
but wihout this condition the pushable term is not produced for dblink query.

Reply:
There is same routine at below else statement.
Can't it be handled with that routine?

Reply:
The dblink node might not be a single and the multiple dblink should be processed by while loop with "from->next".

Reply:
If it's multiple dblinks, isn't from->next true?

Reply:
Ok, I understand what you say. I will remove a redundant code.